tests/dialects/test_riscv.py

```diff
@@ -113,7 +113,7 @@ def test_label_op_with_comment():
     assert label_op.label.data == label_str
 
     code = riscv_code(ModuleOp([label_op]))
-    assert code == f"{label_str}:    # my label\n"
+    assert code == f"{label_str}:                                         # my label\n"
```

Comment:
big gap?

Reply:
makes comments be aligned and a bit out of the way; it's just stylistic.